product_docs/docs/pgd/5/rel_notes/pgd_5.5.1_rel_notes.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: "EDB Postgres Distributed 5.5.1 release notes"
+navTitle: "Version 5.5.1"
+---
+
+Released: 31 Mar 2024
+
+EDB Postgres Distributed version 5.5.1 is a minor version bug-fix release of EDB Postgres Distributed.
```

Comment:
It is a patch release containing only bug fixes (not a minor release).

According to Semantic Versioning, it's MAJOR.MINOR.PATCH. This is a bump in the patch part of the version.